### DISCUSSION TITLE
How to set visible of meshes?

### DISCUSSION THREAD

**Discussion by salii15** [google/model-viewer#3918] [Q&A]
Hi, I want to configure chair. I import chair with two type legs. So I want to hide one of them. And set visible other. I want to create set visible of meshes. Anybody know how to do that?

![image](https://user-images.githubusercontent.com/68233176/199318831-d1608ecd-1ef4-46e2-a68a-1e62ee8182e1.png)

**Comment by elalish** [comment]
I'm hoping to make this easier in future with a new API. For now, you need to select the different legs by the their material and set the alpha to zero to hide them (using `setBaseColorFactor`).
